package.json

```diff
@@ -40,7 +40,7 @@
     "swipe-js-iso": "~2.0.1",
     "test-shape-closeness": "0.0.1",
     "visible-element": "~1.0.1",
-    "zooniverse-geordi-client": "~1.3.7"
+    "zooniverse-geordi-client": "zooniverse/geordi-client#panoptes-client"
```

Comment:
Sorry, i have just seen the other PR  for geordi-client - I remembered already merging one but that was functional tests. Will look now. then will merge and create 1.3.11 then we can update here.